libs/hdf-converters/test/mappers/reverse/xccdf_reverse_mapper.spec.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs';
+import {FromHDFToXCCDFMapper} from '../../../index';
+
+describe('XCCDF Results Revse Mapper', () => {
+  it('Successfully converts HDF into XCCDF-Results', () => {
+    const inputData = fs.readFileSync(
+      'sample_jsons/xccdf_reverse_mapper/sample_input_report/rhel7-results.json',
+      {encoding: 'utf-8'}
+    );
+    const outputTemplate = fs.readFileSync(
+      'src/converters-from-hdf/xccdf/hdf2xccdf-results-template.xml'
+    );
+    const mapper = new FromHDFToXCCDFMapper(
+      inputData,
+      outputTemplate.toString(),
+      true
```

Comment:
if we change the behavior of dateoverride, verify that the default value and this value provided here are still applicable